symfony/messenger/4.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#MESSENGER_ADAPTER_DSN": "amqp://guest:guest@localhost:5672/%2f/messages"
+    },
```

Comment:
Perhaps [post-install-output](https://github.com/symfony/recipes#post-install-output-configurator) would be a good fit for mentioning the `symfony/serializer-pack` to start using built-transport easily.

Reply:
Maybe a new `symfony/messenger-pack` would help to install both.

Reply:
Any phrasing suggestion? :)

Reply:
```txt
<bg=blue;fg=white>              </>
<bg=blue;fg=white> What's next? </>
<bg=blue;fg=white>              </>

  * You're ready to use the Messenger component. You can define you message buses
    or start using the default one right now by injecting the `messenger.bus.default` service 
    or typehinting `Symfony\Component\Messenger\MessageBusInterface` in your classes.
  * If you need to send messages to your broker, you can benefit from the built-in
    amqp transport by: 

    1. Installing the <info>symfony/serializer-pack</>
    2. Uncommenting the <comment> MESSENGER_ADAPTER_DSN</> env var 
       and `framework.messanger.transport.amqp` config
    3. Routing your messages to the amqp sender.

  * <fg=blue>Read</> the documentation at <comment>https://symfony.com/doc/master/components/messenger.html</>
```

(something like that)

Reply:
That's an amazing idea!